Sources/API/V1/yorkie/v1/yorkie.pb.swift

```diff
@@ -1006,13 +1010,17 @@ extension Yorkie_V1_PushPullChangesRequest: SwiftProtobuf.Message, SwiftProtobuf
     try { if let v = self._changePack {
       try visitor.visitSingularMessageField(value: v, fieldNumber: 3)
     } }()
+    if self.pushOnly != false {
```

Comment:
Suggest changing it to 
```swift
if self.pushOnly  {
```
for better readability.

Reply:
I totally agreed. but This code was produced automatically by the protoc. So I can't change it. ^^;